packages/fast-usdc/tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "../../tsconfig.json",
-  "compilerOptions": {},
+  "compilerOptions": {
+    "allowImportingTsExtensions": true,
+    "rewriteRelativeImportExtensions": true
```

Comment:
we don't need these compilerOptions in the other Fast USDC packages?

Reply:
good question.`allowImportingTsExtensions` is already true everywhere, so I'm removing it. 
`rewriteRelativeImportExtensions` is necessary only for published packages (so the dist has `.js` files and imports instead)

Reply:
I shouldn't have made any changes to this package tho. Right now I just want to update unpublished packages. I'll remove that commit and force push.

Reply:
> ... so the dist has `.js` files ...

are those dist files generated with `ts-blank-space`?

or are we publishing the stuff that `tsc` generates? does it have `var` and cjs and such?

Reply:
when there is a `dist` generated, it's with `tsc`. e.g. https://github.com/Agoric/agoric-sdk/blob/85e8b186d36f3a7ed4d83b223414ca2600284693/packages/orchestration/package.json#L14-L16